File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int attempt = 0;
        for(int i = 0; i< nums.length; i++)
        {
          for(int j = 0; j< nums.length; j++)
          {
            if(i == j)
            {}
            else
            {
            attempt = nums[i] + nums[j];
            if(attempt == target)
            return new int[] {i, j};
            }
          }
        }
        return nums;
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        seen = {}
        for i, num in enumerate(nums):
            comp = target - num
            if comp in seen:
                return[i, seen[comp]]
            seen[num] = i